sphinx/source/docs/user_guide/styling.rst

```diff
@@ -1117,7 +1117,7 @@ only with :ref:`axis labels <userguide_styling_axes_labels>`. Supporting LaTeX
 notations in more elements is planned for future releases.
 
 Bokeh uses the MathJax_ library to handle LaTeX markup. To use LaTeX markup
-instead of plain text, use a :class:`~bokeh.models.math_text.MathText` object:
+instead of plain text, use a :class:`~bokeh.models.text.MathText` object:
```

Comment:
The split of `bokeh.models` into sub-modules is purely for maintenance considerations. Users should always (and only) be instructed to import from top-level `bokeh.models`

Reply:
Tho I guess refs like this might need to stay the way they are currently, until we can completely re-arrange the reference guide. Disregard the previous comment, for this PR.